antarest/storage/service.py

```diff
@@ -0,0 +1,65 @@
+from io import BytesIO
+from pathlib import Path
+from typing import List, IO
+
+from antarest.common.custom_types import JSON
+from antarest.storage.web import RequestHandler
+from antarest.storage.web.request_handler import RequestHandlerParameters
+
+
+class StorageService:
```

Comment:
should be the other way around. Business code should be in the service and request handler just calling the service methods